build.gradle

```diff
@@ -19,35 +19,33 @@ buildscript {
       dokkaPlugin: "org.jetbrains.dokka:dokka-gradle-plugin:1.6.0",
       mavenPublishPlugin: "com.vanniktech:gradle-maven-publish-plugin:0.18.0",
       androidx: [
-          annotation: 'androidx.annotation:annotation:1.0.2',
-          core: 'androidx.core:core:1.0.1',
-          fragment: 'androidx.fragment:fragment:1.0.0',
+          fragment: 'androidx.fragment:fragment:1.4.0',
```

Comment:
I'm intentionally staying on the lowest version. See https://twitter.com/Piwai/status/1474410599945805832